cvcl/templates/assignment_detail.html

```diff
@@ -4,13 +4,13 @@
 {% block content %}
     <h1>Assignment: {{ assignment.name }}
         <div class="pull-right">
-            <a class="btn btn-default" href="{% url 'assignments' %}" role="button">Back</a>
+            <a class="btn btn-default" href="{% url 'assignments' %}" role="button">Back
```

Comment:
This is ok but there should be an "Update" button-link for instructors who own the assignment